taichi/ir/snode.h

```diff
@@ -76,6 +76,7 @@ class SNode {
   int64 n{};
   int total_num_bits{}, total_bit_start{};
   int chunk_size{};
+  int num_container_bits;  // for bit_struct and bit_array only
```

Comment:
I got little bit confused here. I guess, when we talk about `num_container_bits` for `bit_struct` and `bit_array`, the 'container' means the number of bits used for containing the custom integer, not the real container of `SNode`. If so, should we use a more specific name here?

Reply:
Good point!! Let me use `containing_type` instead of `container` to avoid confusions.